### PR TITLE
emotion v8 + adjustments for breaking changes

### PR DESCRIPTION
**Pull request by hmeissner**
Update for with-emotion example. Updated emotion packages and adjusted the code for changes listed here: https://medium.com/@tkh44/emotion-8-9f892346d0af

### CODE REVIEW

**Comment by timneutkens** [comment]
@tkh44 Do you mind checking this one 😄

File: examples/with-emotion/package.json [hunk 1]
```diff
@@ -7,13 +7,13 @@
     "start": "next start"
   },
   "dependencies": {
-    "babel-plugin-emotion": "^7.1.0",
-    "emotion": "^7.2.0",
-    "emotion-server": "^7.2.0",
+    "babel-plugin-emotion": "8.0.4",
```
Comment (tkh44) [inline examples/with-emotion/package.json]: These should use `^` so we don't get locked into a minor version number.

**Review by tkh44 — CHANGES_REQUESTED** [review]

File: examples/with-emotion/pages/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import { hydrate, keyframes, css, injectGlobal } from 'emotion'
-import styled from 'react-emotion'
+import { hydrate } from 'emotion'
+import styled, { keyframes, css, injectGlobal } from 'react-emotion'
```
Comment (tkh44) [inline examples/with-emotion/pages/index.js]: Hydrate can be imported from `react-emotion` as well. Sorry I missed this the first time around.

**Review by tkh44 — CHANGES_REQUESTED** [review]

File: examples/with-emotion/package.json (hunk 1 above)
Comment (hmeissner) [reply examples/with-emotion/package.json]: I was actually wondering that, but haven't seen hydrate in the medium post (even though you pointed out that react-emotion exports all of emotion and I've seen it in source) and didn't want to risk it for the off chance that it *should* be imported directly from emotion for some reason :D

File: examples/with-emotion/pages/index.js (hunk 2 above)
Comment (hmeissner) [reply examples/with-emotion/pages/index.js]: I was actually wondering that, but haven't seen hydrate in the medium post (even though you pointed out that react-emotion exports all of emotion and I've seen it in source) and didn't want to risk it for the off chance that it should be imported directly from emotion for some reason :D

**Review by tkh44 — APPROVED** [review]

**Review by timneutkens — APPROVED** [review]
Thanks!

**Pull request merged**